recipes/ecctl/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ecctl" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: go-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/elastic/ecctl/archive/v1.0.1.tar.gz
```

Comment:
Please use the version variable here instead of the hardcoded number. That way the bot can pick up updates automatically.